packages/reactotron-app/App/Shared/CommandToolbar.js

```diff
@@ -153,6 +158,22 @@ class CommandToolbar extends Component {
       const text = apiToMarkdown(this.props.command.payload)
       clipboard.writeText(text)
     } catch (e) {
+      clipboard.writeText(body)
+    }
+  }
+
+  // copy api request to clipboard as cURL
+  handleCopyApiRequestToClipboardAsCurl = event => {
+    event.stopPropagation()
+    const { command } = this.props
+    const { payload } = command
+    const body = dotPath('request.data', payload)
+
+    try {
+      const text = apiRequestToCurl(this.props.command.payload)
+      clipboard.writeText(text)
+    } catch (e) {
+      clipboard.writeText(body)
```

Comment:
This would put the payload of the `post` or `patch` on the clipboard which is not what we want for this function. 

Probably best to have a empty catch instead.

Every time I see this empty catches, I think... oh ya... i need to add a error message or something... then i never do.